src/main/java/com/neatresults/mgnltweaks/json/JsonBuilder.java

```diff
@@ -284,7 +291,7 @@ public String print() {
             }
             return json;
         } catch (JsonProcessingException | RepositoryException e) {
-            // ignore
+			log.error("Failed to generate JSON string", e);
```

Comment:
Not big fan of those when they appear in the log on production w/o really knowing which template is causing them. Could you please change this to debug log if you need it there? thx.